atlp-rwanda/Team-furebo-E-commerce-fn
# ch-implement-notification-toasts-185172196

#### What does this PR do?
This task implements toasts  for new notifications and increase the notification count on the notification bell.

#### Description of Task to be completed?
This Pull Request (PR) aims to enhance the notification feature in the application. The task to be completed is to implement a toast notification popup whenever a user receives a new notification. Currently, when users receive a notification, it is only indicated by an increase in the notification count displayed on the notification bell icon.

#### How should this be manually tested?

1. Ensure that the notification bell icon is visible and the current notification count is displayed.
2. Trigger a new notification to be received in the application. This can be done by performing the action that generates a notification (e.g: create a product, buy a product, etc.).
3. Observe the notification bell icon and the notification count to confirm that the count increases.
4. Additionally, a toast notification should appear on the screen indicating the new notification.
5. Verify that the toast notification provides the necessary information about the new notification.

#### What are the relevant pivotal tracker/Trello stories?
#185172196
